source/robot_model/src/Model.cpp

```diff
@@ -184,6 +184,30 @@ bool Model::check_collision(const state_representation::JointPositions& joint_po
   return false;
 }
 
+// Compute minimum distance
+Eigen::MatrixXd Model::compute_minimum_distance(const state_representation::JointPositions& joint_positions) {
+    Eigen::VectorXd configuration = joint_positions.get_positions();
```

Comment:
Do you need to check if the collisions were loaded here?

Reply:
Yep indeed! I have added a throw exception similar to `check_collision`